### ISSUE TITLE
feature request: Better formatting of help text

### ISSUE THREAD

**Issue by SimplicityGuy** [bw2/ConfigArgParse#47]
Current help text is output very strangely:
```
  --user USER           user to login to foobar with [env var: FOOBAR_USER]
  --password PASSWORD   user password to login to foobar with [env var:
                        FOOBAR_PASSWORD]
```

Please consider providing a formater that formats the help text as such:
```
  --user USER           user to login to foobar with
                        [env var: FOOBAR_USER]
  --password PASSWORD   user password to login to foobar with
                        [env var: FOOBAR_PASSWORD]
```

**Comment by bw2** [comment]
If you could submit a PR on this, that'd be much appreciated.

**Comment by SimplicityGuy** [comment]
Absolutely, I'll put something together.

**Comment by SimplicityGuy** [comment]
I started to look at this last night. It looks like this is the default formatting coming from `argparse`. Specifically, on line 819, there is a pre-existing formatting change (adding a few spaces), and upon return  on line 839, those spaces are replaced by a single space. Seems odd that `argparse` doesn't respect the formatting passed into it.

I tried rewriting the output from `argparse`, and this is working with some limited success. It feels a bit hacky to do so and the right fix really should be in argparse.
